apps/server/src/modules/room-member/do/room-member.do.ts

```diff
@@ -0,0 +1,39 @@
+import { ObjectId } from '@mikro-orm/mongodb';
+import { AuthorizableObject, DomainObject } from '@shared/domain/domain-object';
+import { EntityId } from '@shared/domain/types';
+import { GroupEntity } from '@src/modules/group/entity/group.entity';
+
+export interface RoomMemberProps extends AuthorizableObject {
+	id: EntityId;
+	roomId: ObjectId;
+	userGroup: GroupEntity;
+	createdAt: Date;
+	updatedAt: Date;
+}
+
+export type RoomMemberCreateProps = Pick<RoomMemberProps, 'roomId'>;
+export type RoomMemberUpdateProps = RoomMemberCreateProps;
+
+export class RoomMember extends DomainObject<RoomMemberProps> {
```

Comment:
currently this seems unused?
basically, it should be used everywhere in stead of the entity, so the entity only needs to be used in the repo itself.